package.json

```diff
@@ -54,7 +54,7 @@
     "gulp-rename": "1.2.2",
     "gulp-sourcemaps": "1.6.0",
     "gulp-uglify": "2.0.0",
-    "pleaserc": "^2.3.1",
```

Comment:
haha, I've meant to use this but haven't yet so took it out until I update it again later. Got tired of the peerDep warning from it for enzyme 3.4.x